src/progress/pct/compile.p

```diff
@@ -118,6 +118,10 @@ DEFINE VARIABLE iFileList AS INTEGER    NO-UNDO.
 /* Handle to calling procedure in order to log messages */
 DEFINE VARIABLE hSrcProc AS HANDLE NO-UNDO.
 ASSIGN hSrcProc = SOURCE-PROCEDURE.
+MESSAGE "One more line covered".
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) No :T or :U attribute in string [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.StringAttribute)